aries-site/src/examples/templates/PageHeader/PageHeader.js

```diff
@@ -1,24 +1,67 @@
+import { useContext } from 'react';
 import PropTypes from 'prop-types';
-import { Box, Header, Heading, Paragraph } from 'grommet';
+import {
+  Box,
+  Button,
+  Header,
+  Heading,
+  Menu,
+  Paragraph,
+  ResponsiveContext,
+} from 'grommet';
+import { More } from 'grommet-icons';
 
-export const PageHeader = ({ title, subtitle, actions, ...rest }) => (
-  <Header {...rest}>
-    <Box>
-      <Heading level={1} margin="none" size="small">
-        {title}
-      </Heading>
-      {typeof subtitle === 'string' ? (
-        <Paragraph margin="none">{subtitle}</Paragraph>
-      ) : (
-        <>{subtitle}</>
-      )}
-    </Box>
-    {actions}
-  </Header>
-);
+export const PageHeader = ({
+  title,
+  subtitle,
+  actions,
+  contextualNav,
+  ...rest
+}) => {
+  const size = useContext(ResponsiveContext);
+
+  return (
+    <Header direction="column" align="stretch" gap="small" {...rest}>
+      <Box align="start">{contextualNav}</Box>
+      <Box align="start" direction="row" gap="small" justify="between">
```

Comment:
Do we have any designs/guidance for what happens when the Heading is longer and collides with the actions?

Are there max-widths to be defined at various breakpoints?

Reply:
Great question -- I posed this question to Vicky as something she should explore from the design side. I do think there should be a min-width for the text content (taking precedence over the page-level actions).